HISTORY.rst

```diff
@@ -4,6 +4,7 @@ History
 
 0.37.0 (unreleased)
 -------------------
+
```

Comment:
@juliettelavoie  There's a strange blank lines here and also, your name is missing!

Reply:
thanks I fix it!